explorer/src/lib/components/world-map/WorldMap.svelte

```diff
@@ -36,7 +41,7 @@
   {#each dataset.features as data, index (index)}
     <path d={path(data)} style="fill:url(#vertical-lines)" />
   {/each}
-  {#if nodeLocations.length}
+  {#if nodeLocations && nodeLocations.length}
```

Comment:
I think we can skip the `if` and just have:

```svelte
{#each getUniqueMarkers(nodes)}
```

Also I wouldn't use the index as... index: we can use an expression as:
```js
`${marker.lon}x${marker.lat}`
```
to get unique values from markers.